testing_and_setup/compass/ocean/soma/32km/time_varying_forcing/soma_template.xml

```diff
@@ -0,0 +1,92 @@
+<!--
```

Comment:
I was trying to strip out all of the analysis members and extra stream outputs in the existing soma template. I can apply that template and remove those things from it if that would be better.

Reply:
Renaming file to `time_varying_forcing_template.xml` is probably sufficient for now because templates aren't fully flexible, e.g., you can't just pull out namelist or stream entries from a template as far as I know.

Reply:
I went ahead and applied the existing template (`soma_run.xml` in the `script_configuration_dir`) and was able to get the same namelist and streams files for the forward case.

Reply:
Even better, thanks!